### PR TITLE
UI improvments

### PR DESCRIPTION
**Pull request by prayuj**

### CODE REVIEW

**Comment by sushrut111** [comment]
Hey Can you please resolve the conflicts? Sorry to bother you!

**Comment by sushrut111** [comment]
Great work on the UI. While resolving the conflicts, please merge the new changes as well. Let me know if you want me to do it. The UI looks amazing.
Just one comment, the button to open modal is too small. It would be great if you can make it bigger like how it was before.

**Comment by sushrut111** [comment]
The modal header shows "Form", if you can think of something more informative, please update that.

File: contentScript.js [hunk 1]
```diff
@@ -315,18 +319,16 @@ const createHrSeparator = () => {
...
 
   // parent div for form
   let wrapperDiv = document.createElement("div");
   wrapperDiv.id = "formWrapper";
-  wrapperDiv.style = "position: fixed; background: white; top: 12.5%; width: 75%; left: 12.5%; border: 3px solid #73AD21; overflow:scroll; height: 80%"
 
   // mobile number input field
   let mobileinputid = "data-mob";
-  let mobileInput = createInput(mobileinputid, inputStyles, "number", mobilenumber);
+  let mobileInput = createInput(mobileinputid, "", "number", mobilenumber, 'form-control');
```
Comment (sushrut111) [inline contentScript.js:331]: I love this. Love how you didn't deform everything, thanks.

File: contentScript.js [hunk 2]
```diff
@@ -359,97 +361,91 @@ const createForm = () => {
...
+  let enableautorefreshinput = createInput(enableautorefreshid, "", "checkbox", "", 'form-check-input');
   enableautorefreshinput.checked = enableAutoRefresh;
   let enableautorefreshlabel = createLabel("enableautorefreshlabel", enableautorefreshid, "Enable Auto Refresh on search by district ", textLabelStyles);
   let enableautorefreshWarn = createWarningText("Keep auto refreshing every 2 seconds in search with district. This work only if 'Attempt to book continuosly is selected' and search preference is set to District. Use this option carefully. Your access to cowin portal may be blocked if you send too many request during a short duration.", warnLabelStyles);
 
-
   let timeslotinputid = "timeslotinput";
-  let timeslotinput = createInput(timeslotinputid, inputStyles, "number", timeslotind);
+  let timeSlotSelector = createSelectInput(timeslotinputid, "", timeslotind)
+  let one = createSelectOptions("timeSlot-1", "Slot 1", 1, timeslotind == 1)
```
Comment (sushrut111) [inline contentScript.js:383]: You might as well write the timings for each slots

Comment (sushrut111) [reply contentScript.js:383]: I mean in the select text

Comment (sushrut111) [inline contentScript.js:383]: timeslotind is not a number its a string.

**Review by sushrut111 — CHANGES_REQUESTED** [review]

File: contentScript.js (hunk 1 above)
Comment (prayuj) [reply contentScript.js:331]: Thanks!

File: contentScript.js (hunk 2 above)
Comment (prayuj) [reply contentScript.js:383]: I thought of adding the timings but I am unsure if the slots are fixed. 
If you can confirm the slots are the following, I can add that onto the select fields:
1. 9:00 am to 11:00 am
2. 11:00 am to 1:00 pm
3. 1:00 pm to 3:00 pm
4.  3:00 pm to 5:00 pm

Comment (sushrut111) [reply contentScript.js:383]: Yeah, in most cases these are the slots. We have disclaimed that if these are not the slots, slot 2 will be selected by defualt.

File: manifest.json [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
-            "matches": ["https://selfregistration.cowin.gov.in/*"],
+            "matches": [
+                "https://selfregistration.cowin.gov.in/*"
+            ],
             "js": [
                 "jquery.min.js",
                 "contentScript.js",
-                "shortcut.js"
+                "shortcut.js",
+                "bootstrap.min.js"
```
Comment (sushrut111) [inline manifest.json:15]: move this to top, just after jquery.
Also did you check if this works (including css)?

Comment (prayuj) [reply manifest.json:16]: Can you just confirm the order of the js array

**Comment by prayuj** [comment]
> 
> 
> The modal header shows "Form", if you can think of something more informative, please update that.

Is 'Autofill Input Form' okay?
I've wracked my head and that's the best I can come up with 😅

**Comment by prayuj** [comment]
> 
> 
> Great work on the UI. While resolving the conflicts, please merge the new changes as well. Let me know if you want me to do it. The UI looks amazing.
> Just one comment, the button to open modal is too small. It would be great if you can make it bigger like how it was before.

I have increased the button size to large, if you still feel a bigger one is needed, I am happy to do it.
Also regarding the position of the button, I moved it to the bottom left, is that okay, or should I move it back to where it was?

**Comment by sushrut111** [comment]
> > Great work on the UI. While resolving the conflicts, please merge the new changes as well. Let me know if you want me to do it. The UI looks amazing.
> > Just one comment, the button to open modal is too small. It would be great if you can make it bigger like how it was before.
> 
> I have increased the button size to large, if you still feel a bigger one is needed, I am happy to do it.
> Also regarding the position of the button, I moved it to the bottom left, is that okay, or should I move it back to where it was?

IMO, move it back to top, make it "danger" and even bigger.

**Comment by sushrut111** [comment]
> > The modal header shows "Form", if you can think of something more informative, please update that.
> 
> Is 'Autofill Input Form' okay?
> I've wracked my head and that's the best I can come up with 😅

Duh, Not a big deal, keep it form it nothing!

File: manifest.json [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
     "name": "Cowin Booking",
     "description": "Extensio to automatically fill data on cowin portal to speed up covid vaccine slot booking including captcha",
     "version": "2.1.0",
     "content_scripts": [
         {
-            "matches": ["https://selfregistration.cowin.gov.in/*"],
+            "matches": [
+                "https://selfregistration.cowin.gov.in/*"
+            ],
             "js": [
```
Comment (sushrut111) [inline manifest.json:11]: Order: jquery, bootstrap, contentscript, shortcut, sessionexp

**Comment by prayuj** [comment]
Can you review the changes now @sushrut111 ?

**Comment by sushrut111** [comment]
There was a shortcut, alt + x which opens the modal, it is not working, check shortcut.js file.

File: contentScript.js (hunk 2 above)
Comment (sushrut111) [reply contentScript.js]: did you not check this?

**Review by sushrut111 — APPROVED** [review]

**Comment by sushrut111** [comment]
Great job friend, I love the new look! Thanks for your valuable time!

**Comment by prayuj** [comment]
You already merged it! I was fixing the alt+x

**Comment by sushrut111** [comment]
> You already merged it! I was fixing the alt+x

Please create a new PR. @prayuj

**Comment by prayuj** [comment]
> 
> 
> > You already merged it! I was fixing the alt+x
> 
> Please create a new PR. @prayuj

Okay 👍

**Pull request merged**